$OpenBSD: patch-modules_audio_output_sndio_c,v 1.2 2022/03/02 12:03:17 sthen Exp $

sndio: fix crash when volume is adjusted while stopped
e94b04c8f867382ce6835254d29ee54fe855c686

Index: modules/audio_output/sndio.c
--- modules/audio_output/sndio.c.orig
+++ modules/audio_output/sndio.c
@@ -208,6 +208,8 @@ static void Stop (audio_output_t *aout)
 {
     aout_sys_t *sys = aout->sys;
 
+    aout->volume_set = NULL;
+    aout->mute_set = NULL;
     sio_close (sys->hdl);
 }
 
